cartography/data/jobs/cleanup/gcp_iam_domains_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (:GCPCustomer)-[:HAS_DOMAIN]->(n:GCPDomain) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+            "iterative": true,
+            "iterationsize": 100,
+            "__comment__": "Delete GCP IAM Domains that no longer exist and detach them from all previously connected nodes."
+        },
+        {
+            "query": "MATCH (:GCPCustomer)-[r:HAS_DOMAIN]->(:GCPDomain) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
+            "iterative": true,
+            "iterationsize": 100,
+            "__comment__": "Remove GCP Customer to IAM Domains relationships that are out of date"
+        },
+        {
+            "query": "MATCH (:CloudanixWorkspace)-[:OWNER]->(:GCPProject)-[:RESOURCE]->(n:GCPDomain) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
We should not have any reference of CloudanixWorkspace in our feature branches. That node is very specific to our implementation. Please remove all references of CloudanixWorkspace from IAM code.